### PR TITLE
oskar/help_center

### PR DESCRIPTION
**Pull request by OskarAhl**
Add:

* Help center
* LinkText Component

### CODE REVIEW

File: src/pages/help-centre.js [hunk 1]
```diff
@@ -1,13 +1,688 @@
...
+`
+
+const Article = ({ article, all_articles, onClick, toggleSearch }) => {
+    const related_articles = getRelatedArticles(all_articles, article)
+
+    return (
+        <>
+            <LeftRightContainer padding="8px 0 0 0">
+                <StyledLink to="/help-centre/">
+                    {localize('Back to Help topics')}
```
Comment (arash-binary) [inline src/pages/help-centre.js]: should this one have underline text-decoration?

File: src/pages/help-centre.js [hunk 2]
```diff
@@ -1,13 +1,688 @@
...
+const articles = [
+    {
+        category: localize('Account'),
+        articles: [
+            {
+                title: localize('Who can open an account?'),
+                category: localize('Account'),
+                sub_category: localize('Opening an account'),
+                label: 'who-can-open-an-account',
+                content: <WhoCanOpenAnAccount />,
```
Comment (arash-binary) [inline src/pages/help-centre.js]: All these links have underline text-decoration when you get hover on them. please check it is right or not.

File: src/pages/help-centre.js [hunk 3]
```diff
@@ -1,13 +1,688 @@
...
+`
+const Backdrop = styled.div`
+    background-color: var(--color-black);
+    padding: 12rem 0;
+`
+
+const SearchIconBig = styled(SearchIcon)`
+    position: absolute;
+    left: 0;
+    top: 3px;
```
Comment (arash-binary) [inline src/pages/help-centre.js]: can we use pixel?

File: src/pages/help-centre.js [hunk 4]
```diff
@@ -1,13 +1,688 @@
...
+`
+
+const StyledLink = styled(LocalizedLink)`
+    font-size: 2rem;
+    color: var(--color-red);
+
+    &::before {
+        content: '<';
+        display: inline-block;
+        margin-right: 4px;
```
Comment (arash-binary) [inline src/pages/help-centre.js]: is pixel allowed?

File: src/pages/help-centre.js [hunk 5]
```diff
@@ -1,13 +1,688 @@
...
+        font-weight: 300;
+        text-decoration: none;
+    }
+`
+
+const SmallSearchIcon = styled(SearchIcon)`
+    height: 27px;
```
Comment (arash-binary) [inline src/pages/help-centre.js]: what about here?

File: src/pages/help-centre.js [hunk 6]
```diff
@@ -1,13 +1,688 @@
...
+    width: 27px;
```
Comment (arash-binary) [inline src/pages/help-centre.js]: and here ...

File: src/pages/help-centre.js [hunk 7]
```diff
@@ -1,13 +1,688 @@
...
+
+const getAllArticles = articles =>
+    articles
+        .map(category => category.articles)
+        // flatten the array, gatsby build does not support .flat() yet
+        .reduce((arr, article_arr) => arr.concat(article_arr), [])
+
+const SearchSection = styled.section`
+    ${Backdrop} {
+        max-height: ${props => (props.show ? '1000px' : '0')};
```
Comment (arash-binary) [inline src/pages/help-centre.js]: can this `px` convert to `rem`?

File: src/components/elements/typography.js [hunk 8]
```diff
@@ -17,6 +17,20 @@ export const Text = styled.p`
     ${BaseElement}
 `
 
+export const LinkText = styled.a`
+    /* prettier-ignore */
+    color: var(--color-${props => props.color || 'black'});
+    font-size: ${props => props.size || 'var(--text-size-s)'};
+    line-height: ${props => props.lh || '1.5'};
+    display: block;
```
Comment (cakasuma) [inline src/components/elements/typography.js]: I don't think `display` is needed here?

File: src/pages/help-centre.js [hunk 9]
```diff
@@ -1,13 +1,688 @@
...
-        <h1>{localize('Help centre')}</h1>
-    </Layout>
+const WhoCanOpenAnAccount = () => (
+    <>
+        <Text>
+            {localize(
+                'You may only open an account with us on these conditions:',
+            )}
+        </Text>
+        <Ul>
```
Comment (cakasuma) [inline src/pages/help-centre.js]: `ul`?

File: src/pages/help-centre.js [hunk 10]
```diff
@@ -1,13 +1,688 @@
...
+            <Li>
```
Comment (cakasuma) [inline src/pages/help-centre.js]: `li` ?

File: src/pages/help-centre.js [hunk 11]
```diff
@@ -1,13 +1,688 @@
...
+            this.setState({
+                selected_article,
+                toggle_search: false,
+                search_has_transition: false,
+            })
+        }
+    }
+
+    componentDidUpdate = () => {
+        const current_label = location.hash ? location.hash.substring(1) : ''
```
Comment (cakasuma) [inline src/pages/help-centre.js]: we should make a utility function of this i guess

File: src/pages/help-centre.js (hunk 3 above)
Comment (oskar-binary) [reply src/pages/help-centre.js]: For fixed values we should use pixels yes (if it's not supposed to scale across screen sizes)

File: src/pages/help-centre.js (hunk 4 above)
Comment (cakasuma) [inline src/pages/help-centre.js]:
```suggestion
        margin-right: 0.4rem;
```

File: src/pages/help-centre.js [hunk 12]
```diff
@@ -1,13 +1,688 @@
...
+            </ListWrapper>
+        ))}
+    </>
+)
+ArticleList.propTypes = {
+    articles: PropTypes.array.isRequired,
+    onClick: PropTypes.func,
+}
+
+const ListNoBullets = styled.ul`
```
Comment (cakasuma) [inline src/pages/help-centre.js]: can we moved styled components on top ?

File: src/pages/help-centre.js (hunk 1 above)
Comment (oskar-binary) [reply src/pages/help-centre.js]: Ah correct thanks it should have underline when it's hovered only

File: src/pages/help-centre.js [hunk 13]
```diff
@@ -1,13 +1,688 @@
...
+    margin-bottom: 4.2rem;
+    list-style: none;
+
+    > *:not(:last-child) {
+        padding-bottom: 1.6rem;
+    }
+`
+
+const LinkList = ({ list, onClick, link_style }) => (
```
Comment (cakasuma) [inline src/pages/help-centre.js]: reminded me of `LinkedList` C programming 😂

Comment (OskarAhl) [reply src/pages/help-centre.js]: `ListWithLinks` better?

File: src/pages/help-centre.js (hunk 2 above)
Comment (OskarAhl) [reply src/pages/help-centre.js]: Yes should have

**Pull request merged**